addons/ui/fnc_initDisplayInventory.sqf

```diff
@@ -228,44 +231,54 @@ _uniformItems setVariable [QGVAR(containerType), "UNIFORM_CONTAINER"];
 
-{
-    _x ctrlAddEventHandler ["lbDblClick", {
-        params ["_control", "_index"];
-        private _unit = call CBA_fnc_currentUnit;
+private _controls = [
+    _groundItems, _containerItems,
+    _uniformItems, _vestItems, _backpackItems
+];
+_display setVariable [QGVAR(controls), _controls];
 
-        private _container = objNull;
-        private _containerType = _control getVariable QGVAR(containerType);
-        switch _containerType do {
-            case "GROUND": {
-                _container = GVAR(CurrentGroundItemHolder);
-            };
-            case "CARGO": {
-                _container = GVAR(CurrentContainer);
-            };
-            case "UNIFORM_CONTAINER": {
-                _container = uniformContainer _unit;
-            };
-            case "VEST_CONTAINER": {
-                _container = vestContainer _unit;
-            };
-            case "BACKPACK_CONTAINER": {
-                _container = backpackContainer _unit;
-            };
-        };
+// Item Context Menu Framework - lbDblClick
+if !(isNil QGVAR(ItemContextMenuOptions)) then {
+    {
+        _x ctrlAddEventHandler ["lbDblClick", {
+            params ["_control", "_index"];
+            ([_control, _index] call FUNC(getInventoryItemData)) params ["_classname", "_container", "_containerType"];
+            [ctrlParent _control, _container, _classname, _containerType] call FUNC(openItemContextMenu);
+        }];
+    } forEach _controls;
+};
 
-        // Reports classname, but only for magazines.
-        private _classname = _control lbData _index;
-        if (_classname isEqualTo "") then {
-            // For weapons, items and glasses, use the lb index and compare with cargo item list.
-            private _cargoItems = weaponCargo _container + itemCargo _container + magazineCargo _container;
-            _cargoItems = _cargoItems arrayIntersect _cargoItems;
+// Item Rename Framework
+if !(isNil QGVAR(renamedItems)) then {
+    private _fnc_updateInventoryControls = {
+        params ["_display"];
 
-            _classname = _cargoItems param [_index, ""];
-        };
+        {
+            private _control = _x;
 
-        [ctrlParent _control, _container, _classname, _containerType] call FUNC(openItemContextMenu);
-    }];
-} forEach [
-    _groundItems, _containerItems,
-    _uniformItems, _vestItems, _backpackItems
-];
+            if (ctrlShown _control) then {
+                for [{_i = 0}, {_i < (lbSize _control)}, {_i = _i + 1}] do {
+                    private _classname = ([_control, _i] call FUNC(getInventoryItemData)) select 0;
+                    if !(isNil "_classname") then {
+                        private _renameParams = GVAR(renamedItems) getVariable [_classname, []];
+
+                        if !(_renameParams isEqualTo []) then {
+                            _renameParams params ["_name", "_picture"];
+
+                            if !(_name isEqualTo "") then {
```

Comment:
I prefer `!=`, but at least update to `isNotEqualTo`:
```suggestion
                            if (_name != "") then {
```

Update where necessary.